simple/stats/stat_var_hierarchy_generator.py

```diff
@@ -47,9 +47,25 @@ def _generate_internal(triples: list[Triple]) -> "StatVarHierarchy":
   svgs = _create_all_svgs(svs)
   # Sort by SVG ID so it's easier to follow the hierarchy.
   svgs = dict(sorted(svgs.items()))
+  # Create hierarchy root SVG.
```

Comment:
hmm, why not directly under dc/g/Root?

Reply:
I'm fine putting it directly under `dc/g/Root`. Was putting it under a different root following an earlier discussion: https://github.com/datacommonsorg/import/pull/295#discussion_r1556161136

Let me know your preference.

Reply:
Chatted offline - decided to put hierarchy under custom dc root `c/g/Root` and always create one if generating sv hierarchy.

Note that the dcid was changed from `custom/g/Root` to `c/g/Root` to follow convention and goldens have changed accordingly.